.github/ai-automation.yml

```
@@ -0,0 +1,138 @@
+               - Check for adherence to project conventions (ECS, Yuka).
+               - Provide constructive feedback.
+               
+            3. Use available tools to read files and post comments.
+
+  # ------------------------------------------------------------------
+  # AUTOMATED ISSUE TRIAGE (Gemini)
+  # ------------------------------------------------------------------
+  issue-triage:
+    if: github.event_name == 'issues'
+    runs-on: ubuntu-latest
+    if: github.event_name == 'issues'
```

YAMLlint (1.37.1)

[error] duplication of key "if" in mapping

(key-duplicates)